### PR TITLE
Help container component

### PR DESCRIPTION
**Pull request by jonmattgray**
A help component with fixed left/right tail
image

image

### CODE REVIEW

File: src/HelpDiv/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import styled from 'styled-components'
+
+export interface HelpDivProps {
+  background?: string
+  padding?: string
+  margin?: string
+  width?: string
+  height?: string
+  borderRadius?: string
+  leftOrRight?: string
```
Comment (mattdean-digicatapult) [inline src/HelpDiv/index.tsx:11]: being picky. Maybe go with 

```
position: 'bottomLeft' | 'bottomRight'
```

File: src/HelpDiv/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    bottom: -0.7em;
+    border: 0.8em solid ${({ background }) => background};
+    rotate: 45deg;
+    border-bottom-right-radius: 0.8em;
+    box-shadow: 4px 4px 5px rgba(0, 0, 0, 0.1);
+    ${({ leftOrRight }) => leftOrRight}: 2em;
+  }
+`
+
+export default HelpDiv
```
Comment (mattdean-digicatapult) [inline src/HelpDiv/index.tsx:64]: Can we go with another name. Like `TooltipContainer` or `HelpContainer`? I just don't ike `Div`

Comment (jonmattgray) [reply src/HelpDiv/index.tsx:64]: I'll go with `HelpContainer` since tooltip suggests it's hover based

File: src/HelpDiv/index.tsx (hunk 1 above)
Comment (jonmattgray) [reply src/HelpDiv/index.tsx]: went with `tailPosition` since `position` is already a css property

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**